service/resource/legacyv1/resource.go

```diff
@@ -22,8 +22,8 @@ import (
 	awsutil "github.com/giantswarm/aws-operator/client/aws"
 	"github.com/giantswarm/aws-operator/resources"
 	awsresources "github.com/giantswarm/aws-operator/resources/aws"
-	"github.com/giantswarm/aws-operator/service/cloudconfig"
-	"github.com/giantswarm/aws-operator/service/key"
+	"github.com/giantswarm/aws-operator/service/cloudconfigv1"
+	"github.com/giantswarm/aws-operator/service/keyv1"
 )
 
 const (
```

Comment:
The Github UI is weird so I cannot comment on the lines below. Just want to point out that the resources have names in the constants block as well and they also have to be aligned with the version naming.